app/controllers/members_controller.rb

```diff
@@ -7,9 +7,20 @@ class MembersController < ApplicationController
 
   def index
     @sort = params[:sort]
-    @members = members
+
+    order = if @sort == 'recently_joined'
```

Comment:
You could factor this out as a method, which should make CodeFactor happy again.